src/__tests__/App.test.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import App from "../components/App";
+
+describe("App", () => {
+    const { asFragment } = render(<App />);
+
+    it("renders correctly", () => {
+        expect(asFragment()).toMatchSnapshot();
+    });
+});
```

Comment:
I would move your render inside of the test, i.e. 
```
it("renders correctly", () => {
    const { asFragment } = render(<App />);

    expect(asFragment()).toMatchSnapshot();
});
```
This is because, if you were to add more tests, you may want different test setups, and you want to isolate tests so that they don't have unexpected consequences on each other.